### ISSUE TITLE
missing x, y, z counts from pmfe-findmfe output

### ISSUE THREAD

**Issue by ceheitsch** [gtDMMB/pmfe2023#5]
@maxieds I recently ran the following:
```./pmfe-findmfe o.nivara_tRNA.fasta``` from the command line in the ```GTDMMBSoftware2020/pmfe/bin``` directory.
The output was 
> (((((((><((((........))))((((((.......))))))>..<(((((.......))))))))))))>	-321/10	-271/10
which is correct but incomplete.  It should be
> (((((((><((((........))))((((((.......))))))>..<(((((.......))))))))))))>   1  6  4   -321/10	-271/10
since the <x,y,z,w> vector for this structure is <1,6,4,-321/10>.

**Comment by ceheitsch** [comment]
Noting that the problem seems to resolve itself fairly quickly:  `pmfe-findmfe` was run four times on the sequence  d.5.a.P.aerophilum1I_6D9.fasta with parameters `-b 0 -d 1 -a 7.6 -c -1` in `-I -O` mode.  

Note the different formatting on the signatures in attached screen shot:
<img width="1228" alt="Screen Shot 2023-11-15 at 8 25 47 AM" src="https://github.com/gtDMMB/pmfe2023/assets/43553267/ef874b6a-5b88-4479-a0fe-36360bda2015">

However, when cut and pasted from the command line the output reads as:
>	1-423/5	-77
>	1	6	0	-423/5	-77
>	1	6	0	-423/5	-77
> 	1	6	0	-423/5	-77
In other words, the differences between the 2nd, 3rd, and 4th ones disappear.

The sequence is GCCCCUAACCCGGCCAUAGGCGCCGGUGAUACGCCCGGUCUCAUCAGAACCCGGAAGCUAAGGCCGGCGCCGCGCUCGGGAGUACUGGGCUCCGCGAGGGCCCGGGAAACCGGCGUGCUGGGAGGGGGCUU

**Comment by maxieds** [comment]
@ceheitsch 
Follow-up when you can.

### Issues to hack on code 

I am not able to debug without a laptop to login to the mathmulberry server. My GTMath account access ends after the end of this year. Are you going to want me to work on this enough to get me access to hack on the code? My email has changed: maxieds-hackercat(AT)outlook(DOT)com. Have you migrated to BlueSky, or are you still able to communicate with those of us in the Central, USA, North America, Earth timezone - default existential plane, level zero? Please make sure to follow-up any email correspondence with a reference here. Recall while I scream *Adult Swim* and other objectionable four and six-letter conjugated obscenities that the Republicans are not innocent, sweet little children (especially seeing first hand the chaos of martial law coming out of Florida this year into UNICEF at the ATL Jackson-Hartsfield airport right after Trump was indicted in Manhattan). 

I am having very poor reliability getting email messages through to any account. GMail is an impossible situation due to Eric Trump's de facto control over Google and the recent [AI deployment beast unleashed on networks abroad](https://github.com/maxieds/public-file-stashes/blob/main/PROJECTS.md#archive-and-enumerate-new-arvr) (see also [this link](https://github.com/maxieds/public-file-stashes/blob/markdown-drafts/_project/AGO-COMPONENT-RADIO-EXTNET.md) and [this one](https://github.com/maxieds/public-file-stashes/blob/markdown-drafts/_project/AGO-COMPONENT-TIMEZONES.md) and of course the [post pinned to my Github profile here](https://github.com/maxieds/ChameleonMiniLiveDebugger/issues/26#issuecomment-1722385007)). Notably the following warning has been reiterated: *Mitigating the risk of extinction from AI should be a global priority alongside other societal risks such as pandemics and nuclear war.* These network connectivity phenomena are not isolated and caste-like groups of Internet-enabled users (e.g., me) with AndroidGo and other newer TLD (like Google's hostile takeover of the `.app` top-level-domains) and access mechanisms depending on everything from which URL/URI link entered into a site for access to device type platforms like desktop/mobile/web and other FUBAR'ed access control mechanisms that obey neither logic nor reason. While daddy squirms on state-level charges before the next election year, the situation gets farther out of hand for many. You have heard of `chimera / chimerax`, correct? That scary functionality is a component of the new Android(Go) platform. Google should be going through antitrust litigation the next few years. In the meantime, we have war in Europe, the Middle East and (depending on who you ask) at our Southern border in this country (USA). I think that covers the problems at hand pretty well.

### Speculative solutions for you to look at

The directory you are using to invoke the PMFE binary is setup [according to these docs](https://github.com/gtDMMB/pmfe/blob/math-mulberry-testing/MATH-MULBERRY.md). I do not know yet how much the source code has changed in this repository. That will definitely be a factor. Recall that there were a number of modifications and work arounds necessary to get the binaries to build with local `libBoost` and `libCGAL` versions. My first intuition is that you have different versions of the shared/dynamic libraries linked against differently compiled versions of the binaries. Ruling that out first is the first step. Try to pin down which libraries are involved (command syntax differs on MacOS):
```bash
## Get absolute path to binary
$ export binExePath=`readlink -f pmfe-bin-name`
## List shared libraries (multiple ways):
$ objdump -p $binExePath | grep 'NEEDED'
$ ldd $binExePath | grep 'NEEDED'
$ readelf -d $binExePath | grep 'NEEDED'
$ strace -e trace=open,openat $binExePath
# Or on MacOS:
$ otool -L $binExePath
## Parse the list of library paths (basename + file extension):
$ locate libname.so.1.0.0
```
Also recall that [in this patched branch](https://github.com/gtDMMB/pmfe/tree/CustomBuildScriptMods-Summer2022-v1/iB4e), there were some manual modifications needed to get the CGAL headers to work.

**Comment by maxieds** [comment]
@ceheitsch 
How are you accessing the directory `GTDMMBSoftware2020/pmfe`? I can login to the GT math gateway by ssh. Then `ssh math-mulberry` will not resolve the old host where I could find the projects directory.
